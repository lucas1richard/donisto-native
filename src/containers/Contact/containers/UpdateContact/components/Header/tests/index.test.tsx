import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import UpdateContactHeader from '..';

describe('<UpdateContactHeader />', () => {
  const renderComponent = () => toJson(shallow(<UpdateContactHeader />));
  it('renders with just text', () => {
    expect(renderComponent()).toMatchSnapshot();
  });
});