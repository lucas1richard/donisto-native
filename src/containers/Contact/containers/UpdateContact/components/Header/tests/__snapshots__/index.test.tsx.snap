// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UpdateContactHeader /> renders with just text 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <UpdateContactHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <TouchableOpacity
          activeOpacity={0.2}
          onPress={[Function]}
        >
          <Component>
            <Icon
              allowFontScaling={false}
              color="rgb(251, 100, 65)"
              name="chevron-left"
              size={30}
            />
          </Component>
        </TouchableOpacity>,
        <Txt
          color="default"
          style={
            Object {
              "fontSize": 22,
              "fontWeight": "600",
            }
          }
        >
          Update Contact
        </Txt>,
        <Component
          style={
            Object {
              "width": 25,
            }
          }
        />,
      ],
      "style": Object {
        "alignItems": "center",
        "borderBottomColor": "#d3d3d3",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "space-between",
        "paddingHorizontal": 20,
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "activeOpacity": 0.2,
          "children": <Component>
            <Icon
              allowFontScaling={false}
              color="rgb(251, 100, 65)"
              name="chevron-left"
              size={30}
            />
          </Component>,
          "onPress": [Function],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <Icon
              allowFontScaling={false}
              color="rgb(251, 100, 65)"
              name="chevron-left"
              size={30}
            />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "allowFontScaling": false,
              "color": "rgb(251, 100, 65)",
              "name": "chevron-left",
              "size": 30,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "Update Contact",
          "color": "default",
          "style": Object {
            "fontSize": 22,
            "fontWeight": "600",
          },
        },
        "ref": null,
        "rendered": "Update Contact",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "style": Object {
            "width": 25,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <TouchableOpacity
            activeOpacity={0.2}
            onPress={[Function]}
          >
            <Component>
              <Icon
                allowFontScaling={false}
                color="rgb(251, 100, 65)"
                name="chevron-left"
                size={30}
              />
            </Component>
          </TouchableOpacity>,
          <Txt
            color="default"
            style={
              Object {
                "fontSize": 22,
                "fontWeight": "600",
              }
            }
          >
            Update Contact
          </Txt>,
          <Component
            style={
              Object {
                "width": 25,
              }
            }
          />,
        ],
        "style": Object {
          "alignItems": "center",
          "borderBottomColor": "#d3d3d3",
          "borderBottomWidth": 1,
          "flexDirection": "row",
          "height": 60,
          "justifyContent": "space-between",
          "paddingHorizontal": 20,
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "activeOpacity": 0.2,
            "children": <Component>
              <Icon
                allowFontScaling={false}
                color="rgb(251, 100, 65)"
                name="chevron-left"
                size={30}
              />
            </Component>,
            "onPress": [Function],
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <Icon
                allowFontScaling={false}
                color="rgb(251, 100, 65)"
                name="chevron-left"
                size={30}
              />,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "allowFontScaling": false,
                "color": "rgb(251, 100, 65)",
                "name": "chevron-left",
                "size": 30,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "Update Contact",
            "color": "default",
            "style": Object {
              "fontSize": 22,
              "fontWeight": "600",
            },
          },
          "ref": null,
          "rendered": "Update Contact",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "style": Object {
              "width": 25,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": false,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
